packaging/source/buildpackage.sh

```diff
@@ -3,17 +3,19 @@
 
 set -o errexit -o pipefail || exit $?
 
-if [ $# -ne "2" ]; then
-	echo "Usage: $(basename "$0") tag outputdir"
-	exit 1
-fi
-
 # Set the working dir to the location of this script
 HERE=$(dirname "$0")
 cd "${HERE}"
 
-TAG="$1"
-OUTPUTDIR="$2"
+TAG="${1:-devtest-19700101}"
```

Comment:
You are hardcoding a random value.

Reply:
This isn't hard-coded: it is a fallback if the value is not supplied. Typically if a date is required for a variable during a reproduceable build scenario, a good choice is the Unix epoch (1 January 1970). This will not affect releases where a tag is supplied.